liblumen_alloc/src/erts/process/alloc/heap_alloc.rs

```diff
@@ -541,6 +531,64 @@ pub trait HeapAlloc {
         // Return box to tuple
         Ok(Term::make_boxed(tuple_ptr))
     }
+
+    /// Constructs a `Closure` from a slice of `Term`
+    ///
+    /// Be aware that this does not allocate non-immediate terms in `elements` on the process heap,
+    /// it is expected that the slice provided is constructed from either immediate terms, or
+    /// terms which were returned from other constructor functions, e.g. `make_binary_from_str`.
+    ///
+    /// The resulting `Term` is a box pointing to the closure header, and can itself be used in
+    /// a slice passed to `closure_with_env_from_slice` to produce nested closures or tuples.
+    fn closure_with_env_from_slice(
+        &mut self,
+        mfa: Arc<ModuleFunctionArity>,
+        code: Code,
+        creator: Term,
+        slice: &[Term],
+    ) -> Result<Term, Alloc> {
+        self.closure_with_env_from_slices(mfa, code, creator, &[slice])
+    }
+
+    /// Constructs a `Closure` from slices of `Term`
+    ///
+    /// Be aware that this does not allocate non-immediate terms in `elements` on the process heap,
+    /// it is expected that the slice provided is constructed from either immediate terms, or
+    /// terms which were returned from other constructor functions, e.g. `make_binary_from_str`.
```

Comment:
`make_binary_from_str` is called `binary_from_str`.  The tuple comments have the old name.